tests/generic/bug01390-001.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+Test for PHP-1390: Seriously?
```

Comment:
Although I agree with the sentiment, I don't think that should be our test case name :-)